src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet link references a `.scss` file. SCSS files need to be compiled to CSS before they can be used in an HTML document. Consider compiling `styles/main.scss` to `styles/main.css` and updating the link accordingly.